pkgs/development/perl-modules/Bio-BigFile/default.nix

```
@@ -0,0 +1,75 @@
+{ lib, perl, buildPerlModule, perlPackages, fetchurl, fetchFromGitHub, pkgs }:
+# This version of bio::DB::bigfile is suited for ENSEMBL VEP
+# See https://www.ensembl.org/info/docs/tools/vep/script/vep_download.html#bigfile
+
+# Even though there is a kent package on nix, we have to override it to an older version
+let
+  kent335 = pkgs.kent.overrideAttrs (old: rec {
+    version = "335";
+    name = "kent" ;
+
+    src = fetchFromGitHub {
+      owner = "ucscGenomeBrowser";
+      repo = name;
+      rev = "v${version}_base";
+      sha256 = "1455dwzpaq4hyhcqj3fpwgq5a39kp46qarfbr6ms6l2lz583r083";
+    };
+    # Use the ENSEMBL instructions, with some changes from
+    # https://github.com/NixOS/nixpkgs/blob/nixos-22.05/pkgs/applications/science/biology/kent/default.nix#L70
+    # We *cannot* use the package from nixpkgs (tests fails)
+    buildPhase =
+      ''
+      export MACHTYPE=$(uname -m)
+      export CFLAGS="-fPIC"
+      export MYSQLINC=`mysql_config --include | sed -e 's/^-I//g'`
+      export MYSQLLIBS=`mysql_config --libs`
+
+      cd ./src/lib
+      echo 'CFLAGS="-fPIC"' > ../inc/localEnvironment.mk
+
+      # Build kent source
+      make clean && make
+      cd ../jkOwnLib
+      make clean && make
+      mkdir -p $NIX_BUILD_TOP/bin/$MACH_TYPE
+      mkdir -p $NIX_BUILD_TOP/lib
+
+      cp ../lib/x86_64/jkOwnLib.a $NIX_BUILD_TOP/lib
+      cp ../lib/x86_64/jkweb.a $NIX_BUILD_TOP/lib
+
+      # Modified: add include files
+      cp -r ../inc $NIX_BUILD_TOP/
+      '';
+
+      # Only install lib and include. Lib must have the architecture
+      installPhase = ''
+      mkdir -p $out/lib/x86_64
+      cp $NIX_BUILD_TOP/lib/jkOwnLib.a $out/lib/x86_64/
+      cp $NIX_BUILD_TOP/lib/jkweb.a $out/lib/x86_64/
+      cp -r $NIX_BUILD_TOP/inc $out/
+      '';
+    });
+in
+
+buildPerlModule rec {
+  pname = "Bio-BigFile";
+  version = "1.07";
+
+  src = fetchurl {
+    url = "mirror://cpan/authors/id/L/LD/LDS/Bio-BigFile-${version}.tar.gz";
+    sha256 = "277b66ce8acbdd52399e2c5a0cf4e3bd5c74c12b94877cd383d0c4c97740d16d";
+  };
+  buildInputs = (with pkgs; with perlPackages; [ BioPerl IOString kent335 libpng libmysqlclient openssl ]);
```

```suggestion
  };

  buildInputs = [ BioPerl IOString kent335 libpng libmysqlclient openssl ];
```
pkgs should generally tried to be avided as much as possible and perlPackages should not be used inside of perlPackages.